helm/app-operator/templates/deployment.yaml

```diff
@@ -37,9 +45,11 @@ spec:
         volumeMounts:
         - name: {{ include "name" . }}-configmap
           mountPath: /var/run/{{ include "name" . }}/configmap/
+        {{- if not .Values.initialBootstrapMode }}
```

Comment:
Because of the hostNetwork: true to hostNetwork: false switch.

Once you run a deployment in hostNetwork, the `Ports` section get a `hostPort` field set.
Once you change to set hostNetwork = false, for some reason the `hostPort` field (that should go away) stays and makes the replicaset to fail.

This is the easiest workaround but I can workout something else if that's a blocker

Reply:
I am satisfied with the explanations, just maybe adding some short comments would be nice? I am afraid that 3 month from now someone is going to look at this and thing "well why do we exclude this" and create a bunch of work.